hypercertMeasurement.json

```diff
@@ -0,0 +1,55 @@
+{
+  "lexicon": 1,
+  "id": "app.certified.hypercert.measurement",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "External measurement data supporting a hypercert claim",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["hypercert", "measurerDID", "metric", "value", "createdAt"],
+        "properties": {
+          "hypercert": {
+            "type": "ref",
+            "ref": "com.atproto.repo.strongRef",
```

Comment:
Shouldn't this be `app.certified.hypercert.record`? (or whatever we decide that NSID ends up being)